examples/exp_configs/non_rl/aimsun_template.py

```diff
@@ -0,0 +1,58 @@
+"""Load an already existing Aimsun template and run the simulation."""
+from flow.core.params import AimsunParams, EnvParams, NetParams
+from flow.core.params import VehicleParams
+from flow.core.params import InFlows
+import flow.config as config
+from flow.envs import TestEnv
+from flow.networks import Network
+import os
+
+# inflow rate at the highway
+FLOW_RATE = 2000
```

Comment:
is this used? delete if not

Reply:
Good catch! Thanks

Reply:
Oh I forgot to push the commit for removing this. I will include it in the next PR soon.